_team/bill-mcdonald.md

```diff
@@ -3,6 +3,8 @@ title: "Bill McDonald"
 date: 2018-11-19T10:47:58+10:00
 draft: false
 image: "images/team/nonsap-visuals-kMJp7620W6U-unsplash.jpg"
+image_width: 60
```

Comment:
Adding new values for width and height in the front-matter is a bit verbose.

For the team page, I'd probably just set the width and height of 60px in the HTML. It's a pretty decent default.

Reply:
Done!